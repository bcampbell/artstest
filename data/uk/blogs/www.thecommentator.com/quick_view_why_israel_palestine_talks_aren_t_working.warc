WARC/1.0
Content-Length: 13601
Content-Type: application/http; msgtype=response
Warc-Date: 2014-04-24T02:44:36Z
Warc-Record-Id: urn:X-scrapeomat:1398307477770180990
Warc-Target-Uri: http://www.thecommentator.com/article/4887/quick_view_why_israel_palestine_talks_aren_t_working
Warc-Type: response
X-Scrapeomat-Srcurl: http://www.thecommentator.com/article/4887/quick_view_why_israel_palestine_talks_aren_t_working

HTTP/1.1 200 OK
Cache-Control: must-revalidate, private, max-age=0
Content-Type: text/html; charset=utf-8
Date: Thu, 24 Apr 2014 02:44:37 GMT
Etag: "5155248fb9c294de4c129d8c104663c4"
Server: Apache/2.2.22 (Ubuntu)
Status: 200
Vary: Accept-Encoding
X-Powered-By: Phusion Passenger (mod_rails/mod_rack) 3.0.17
X-Rack-Cache: miss
X-Request-Id: a5a8ed401161e67443e03d5243c483f4
X-Runtime: 0.030073
X-Ua-Compatible: IE=Edge,chrome=1

<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta content='width=device-width; initial-scale=1.0; maximum-scale=1.0; user-scalable=0;' name='viewport' />    
    
    <title>Quick View: Why Israel-Palestine talks aren&#x27;t working - The Commentator</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="DnuSNIOwuL72/B9VV/G+X2auvGsjHbpwDGYR5p1FN3Q=" name="csrf-token" />

    <link href="/assets/mobile/base-6bc6617c3340385ceb9812b510622c8e.css" media="all" rel="stylesheet" type="text/css" />
    
    <link href="http://www.thecommentator.com" rel="alternate" title="RSS" type="application/rss+xml" />
	<link rel="alternate" type="application/rss+xml" title="RSS" href="http://www.thecommentator.com/main/rss" />
  <script type="text/javascript">var NREUMQ=NREUMQ||[];NREUMQ.push(["mark","firstbyte",new Date().getTime()]);</script></head>
  <body>
	
		<div class="navbar navbar-inverse navbar-fixed-top">
      <div class="navbar-inner">
        <div class="container">
          <button type="button" class="btn btn-navbar collapsed" data-toggle="collapse" data-target=".nav-collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          
          <div class="nav-collapse collapse" style="height: 0px; ">
            <ul class="nav">
              <li>
                <a href="/">Home</a>
              </li>
              <li>
                <a href="/about">About</a>
              </li>
              <li>
                <a href="/articles/news">News</a>
              </li>
              <li>
                <a href="/articles/commentary">Comment</a>
              </li>
              <li>
                <a href="/articles/tea_room">The Tea Room</a>
              </li>
              <li>
                <a href="http://itunes.apple.com/ie/podcast/we-need-to-talk/id549807532?mt=2" target="_blank">Podcast</a>
              </li>
              <li>
                <a href="/contact">Contact</a>
              </li>
              <li>
                <a href="/desktop_site">Desktop website</a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
	
    <div class="header">
	    <div class="container">
			<div class="row">
				<div class="span12">
					<img alt="Logo" src="/assets/logo-8e3ed70320ba70bf41d0354946382866.png" />
				</div>
			</div>
		</div>
	</div>
	
	<div class="container">

	    			
		<h1>Quick View: Why Israel-Palestine talks aren&#x27;t working</h1>	
		<h4>One of the problems over the Israel-Palestine debate is that many people hold very strong views -- usually against Israel -- without having a clue what they&#x27;re talking about</h4>
		<h5>By <a href="/author/robin_mitchinson/341">Robin Mitchinson</a></h5>
		<p class="date">On 19 April 2014 12:40</p>
		<img alt="Crowds_at_western_wall2_tb_n060601" src="/system/articles/inner_pictures/000/004/887/regular/Crowds_at_Western_Wall2_tb_n060601.jpg?1397911558" />
		<p class="caption">Jerusalem</p>

		  <!-- AddThis Button BEGIN -->
<div class="addthis_toolbox addthis_default_style" addthis:url="http://www.thecommentator.com/article/4887/quick_view_why_israel_palestine_talks_aren_t_working">
  <a class="addthis_button_facebook_like" fb:like:action="recommend" fb:like:layout="button_count"></a>
  <a class="addthis_button_tweet" tw:via="TheCommentator"></a>
  <a class="addthis_button_google_plusone" g:plusone:size="medium"></a>
  <a class="addthis_counter addthis_pill_style"></a>
</div>
<!-- AddThis Button END -->   






					
		<div class="clear" style="height: 20px"></div>
		
		<div id="article_body">

			
				<p id="yui_3_13_0_1_1397910910643_2192">
	It is doubtful whether the contemptible academics who wish to boycott Israel, along with other groups hiding their anti-Semitism behind a charade of being &lsquo;pro-Palestine&rsquo;, have much understanding of what the conflict is all about.</p>
<p id="yui_3_13_0_1_1397910910643_2196">
	The average person is also baffled by the complexity of this never-ending saga. In trying to untangle the issues, the beginning is a good place to start. There is a common fallacy that the Balfour Declaration in 1917 triggered the eventual creation of Israel. Not quite.</p>
<p>
	Here is what it actually said:</p>
<p id="yui_3_13_0_1_1397910910643_2200">
	<em>His Majesty&#39;s government view with favour the establishment in Palestine of a national home for the Jewish people, and will use their best endeavours to facilitate the achievement of this object, it being clearly understood that nothing shall be done which may prejudice the civil and religious rights of existing non-Jewish communities in Palestine, or the rights and political status enjoyed by Jews in any other country</em></p>
<p id="yui_3_13_0_1_1397910910643_2203">
	Note that it speaks of a &lsquo;home&rsquo;, not a state, (and also of the protection of minority rights). But it certainly led to an upsurge in Zionism and Jewish immigration into Palestine. This led to violence and terrorism between Jews and Arabs, with the unfortunate British trying to keep the peace under their League of Nations mandate, and so being attacked by both sides.</p>
<p id="yui_3_13_0_1_1397910910643_2205">
	Statehood had to wait until 1947 when the UN resolved that &nbsp;&ldquo;Independent Arab and Jewish States, and the Special International Regime for the City of Jerusalem shall come into existence in Palestine.&rdquo;</p>
<p id="yui_3_13_0_1_1397910910643_2206">
	So there we have it: the &lsquo;two state&rsquo; solution that the parties are still (occasionally) talking about, plus the designation of Jerusalem as an international city. The way out of the impasse is for all parties simply to adopt Resolution 181.</p>
<p id="yui_3_13_0_1_1397910910643_2207">
	If only it were that easy!</p>
<p id="yui_3_13_0_1_1397910910643_2209">
	Arab countries responded by trying, in three wars, to annihilate Israel. The outcome was that they got thrashed and the Israelis got the Gaza Strip and the land from the West Bank of the River Jordan to the Israeli border plus the whole of Jerusalem.</p>
<p id="yui_3_13_0_1_1397910910643_2212">
	As it stands, the two-state solution would entail Israel giving up its conquests. It went &nbsp;part-way by surrendering Gaza, no great sacrifice. But it continues to populate the West Bank with Jewish settlements. Unsurprisingly, the Knesset has no enthusiasm for the two-state notion, although we have a political dichotomy here; polls suggest that two-thirds of Israelis would support it in some form.</p>
<p id="yui_3_13_0_1_1397910910643_2214">
	What is not going to happen is a return to the original boundaries. This would displace about 600,000 Israelis who have settled in the disputed areas. Neither will Israel accept the &rsquo;right of return&rsquo; demanded by the Palestinian refugees. Sheer numbers would ensure the end of Israel as a Jewish state. Arabs already constitute 20 percent of the population.</p>
<p id="yui_3_13_0_1_1397910910643_2217">
	Creation of a Palestinian state with contiguous boundaries would entail a whole series of land swaps, difficult but perhaps achievable. About 100,000 Jews would be displaced. Gaza does not fit in, and both geography and logic suggest incorporation into Egypt.</p>
<p id="yui_3_13_0_1_1397910910643_2219">
	If there is to be a successful outcome, a great deal of compromise must be deployed. So far this quality has been notably and almost totally absent. Peace may have to await a new generation that has not been conditioned by old horrors and preconceptions.</p>
<p>
	&nbsp;And that&rsquo;s about as simple as I can make it!</p>
<p>
	<em><strong>Robin Mitchinson is a Contributing Editor to <a href="http://www.thecommentator.com/">The Commentator.</a> A former barrister, living in the Isle of Man, he is an international public management specialist with almost two decades of experience in institutional development, decentralisation and democratisation processes. He has advised governments and major international institutions across the world</strong></em></p>

				
			
		</div>

			<h5 class="top_divider">Read more on: <a href="/articles/tag/israel/13672">israel</a>, <a href="/articles/tag/demonising_israel/13800">Demonising Israel</a>, <a href="/articles/tag/israeli_palestinian_coexistance/13930">Israeli-Palestinian coexistance</a>, and <a href="/articles/tag/israel_palestinian_conflict/13495">Israel-Palestinian conflict</a></h5>
			
		  <!-- AddThis Button BEGIN -->
<div class="addthis_toolbox addthis_default_style" addthis:url="http://www.thecommentator.com/article/4887/quick_view_why_israel_palestine_talks_aren_t_working">
  <a class="addthis_button_facebook_like" fb:like:action="recommend" fb:like:layout="button_count"></a>
  <a class="addthis_button_tweet" tw:via="TheCommentator"></a>
  <a class="addthis_button_google_plusone" g:plusone:size="medium"></a>
  <a class="addthis_counter addthis_pill_style"></a>
</div>
<!-- AddThis Button END -->   






		
		
	<div style="text-align:center;">
<script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- Below article -->
<ins class="adsbygoogle"
     style="display:inline-block;width:468px;height:60px;margin: 10px auto;"
     data-ad-client="ca-pub-1814300297407853"
     data-ad-slot="3273227123"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
</div>


	
			<div id="article_comments">
		<div class="heading">
			<span></span>
			Comments
		</div>

		<div id="disqus_thread"></div>
		<script type="text/javascript">
			/* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
			var disqus_shortname = 'commentatorintl'; // required: replace example with your forum shortname
			var disqus_url = 'http://www.thecommentator.com/article/4887/quick_view_why_israel_palestine_talks_aren_t_working';
			var disqus_identifier = '/article/4887/quick_view_why_israel_palestine_talks_aren_t_working';

			/* * * DON'T EDIT BELOW THIS LINE * * */
			(function() {
				var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
				dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
				(document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
			})();
		</script>
		<noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
		<a href="http://disqus.com" class="dsq-brlink">blog comments powered by <span class="logo-disqus">Disqus</span></a>				
	</div>	




	</div>
	
    <div class="footer">
	    <div class="container">
			<div class="row">
				<div class="span12">
					<hr/>
					<a href="#">Back to top</a>
				</div>
			</div>
		</div>
	</div>	
	
    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="/assets/mobile/base-51165128925f9ce4de7e3e87c46f32ed.js" type="text/javascript"></script>
    
    
    <!-- AddThis code -->
	<script type="text/javascript" src="http://s7.addthis.com/js/250/addthis_widget.js#pubid=thecommentator"></script>
	
    <!-- Facebook code -->
	<div id="fb-root"></div>
	<script>(function(d, s, id) {
	  var js, fjs = d.getElementsByTagName(s)[0];
	  if (d.getElementById(id)) return;
	  js = d.createElement(s); js.id = id;
	  js.src = "//connect.facebook.net/en_US/all.js#xfbml=1&appId=454623571228507";
	  fjs.parentNode.insertBefore(js, fjs);
	}(document, 'script', 'facebook-jssdk'));</script>


    <!-- Twitter code -->
	<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src="//platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
    

	<!-- Google Analytics -->
	<script type="text/javascript">

	  var _gaq = _gaq || [];
	  _gaq.push(['_setAccount', 'UA-21015353-1']);
	  _gaq.push(['_trackPageview']);

	  (function() {
		var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
		ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
		var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
	  })();

	</script>
  <script type="text/javascript">if (!NREUMQ.f) { NREUMQ.f=function() {
NREUMQ.push(["load",new Date().getTime()]);
var e=document.createElement("script");
e.type="text/javascript";
e.src=(("http:"===document.location.protocol)?"http:":"https:") + "//" +
  "js-agent.newrelic.com/nr-100.js";
document.body.appendChild(e);
if(NREUMQ.a)NREUMQ.a();
};
NREUMQ.a=window.onload;window.onload=NREUMQ.f;
};
NREUMQ.push(["nrfj","beacon-5.newrelic.com","3c19755cdb",969593,"cQoLRkALW1tdEx1ZFEYMBl5XS0RfVxY=",0,28,new Date().getTime(),"","","","",""]);</script></body>
</html>


