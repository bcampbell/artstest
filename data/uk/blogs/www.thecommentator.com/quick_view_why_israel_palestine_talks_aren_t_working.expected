---
urls:
  - "http://www.thecommentator.com/article/4887/quick_view_why_israel_palestine_talks_aren_t_working"
canonical_url: 
headline: "Quick View: Why Israel-Palestine talks aren't working"
published: 2014-04-19T12:40
authors:
  - name: Robin Mitchinson
publication:
  - name: 
  - domain: www.thecommentator.com
---
<div><p>It is doubtful whether the contemptible academics who wish to boycott Israel, along with other groups hiding their anti-Semitism behind a charade of being ‘pro-Palestine’, have much understanding of what the conflict is all about.</p>
<p>The average person is also baffled by the complexity of this never-ending saga. In trying to untangle the issues, the beginning is a good place to start. There is a common fallacy that the Balfour Declaration in 1917 triggered the eventual creation of Israel. Not quite.</p>
<p>Here is what it actually said:</p>
<p><em>His Majesty&#39;s government view with favour the establishment in Palestine of a national home for the Jewish people, and will use their best endeavours to facilitate the achievement of this object, it being clearly understood that nothing shall be done which may prejudice the civil and religious rights of existing non-Jewish communities in Palestine, or the rights and political status enjoyed by Jews in any other country</em></p>
<p>Note that it speaks of a ‘home’, not a state, (and also of the protection of minority rights). But it certainly led to an upsurge in Zionism and Jewish immigration into Palestine. This led to violence and terrorism between Jews and Arabs, with the unfortunate British trying to keep the peace under their League of Nations mandate, and so being attacked by both sides.</p>
<p>Statehood had to wait until 1947 when the UN resolved that  “Independent Arab and Jewish States, and the Special International Regime for the City of Jerusalem shall come into existence in Palestine.”</p>
<p>So there we have it: the ‘two state’ solution that the parties are still (occasionally) talking about, plus the designation of Jerusalem as an international city. The way out of the impasse is for all parties simply to adopt Resolution 181.</p>
<p>If only it were that easy!</p>
<p>Arab countries responded by trying, in three wars, to annihilate Israel. The outcome was that they got thrashed and the Israelis got the Gaza Strip and the land from the West Bank of the River Jordan to the Israeli border plus the whole of Jerusalem.</p>
<p>As it stands, the two-state solution would entail Israel giving up its conquests. It went  part-way by surrendering Gaza, no great sacrifice. But it continues to populate the West Bank with Jewish settlements. Unsurprisingly, the Knesset has no enthusiasm for the two-state notion, although we have a political dichotomy here; polls suggest that two-thirds of Israelis would support it in some form.</p>
<p>What is not going to happen is a return to the original boundaries. This would displace about 600,000 Israelis who have settled in the disputed areas. Neither will Israel accept the ’right of return’ demanded by the Palestinian refugees. Sheer numbers would ensure the end of Israel as a Jewish state. Arabs already constitute 20 percent of the population.</p>
<p>Creation of a Palestinian state with contiguous boundaries would entail a whole series of land swaps, difficult but perhaps achievable. About 100,000 Jews would be displaced. Gaza does not fit in, and both geography and logic suggest incorporation into Egypt.</p>
<p>If there is to be a successful outcome, a great deal of compromise must be deployed. So far this quality has been notably and almost totally absent. Peace may have to await a new generation that has not been conditioned by old horrors and preconceptions.</p>
<p>And that’s about as simple as I can make it!</p>
<p><em><strong>Robin Mitchinson is a Contributing Editor to<a href="http://www.thecommentator.com/">The Commentator.</a>A former barrister, living in the Isle of Man, he is an international public management specialist with almost two decades of experience in institutional development, decentralisation and democratisation processes. He has advised governments and major international institutions across the world</strong></em></p>
</div>

